types/api-error-handler/api-error-handler-tests.ts

```diff
@@ -1,4 +1,4 @@
-import * as errorHandler from 'api-error-handler';
+import errorHandler = require('api-error-handler');
 import * as express from 'express';
```

Comment:
I'm curious as to why `import * as express` is something valid (for example, in this file) but not in many of the other files

Reply:
We still allow it if it isn't used as a function, which it isn't in this file.